ui/src/main.ts

```diff
@@ -75,7 +73,6 @@ library.add(...icons);
 // let plugins use state mngmnt / router
 (window as any).Vue = Vue;
 (window as any).Pinia = Pinia;
-(window as any).Vuex = Vuex;
```

Comment:
Just double checking - do we have any plugins that use Vuex? I believe Alec only used Pinia. Not sure why we needed this line to begin with.

Reply:
I did think of this and checked Alec, it uses pinia. It's possible it could be an issue with other plugins, but probably best if they don't rely on this and instead handle their own state management.